### DISCUSSION TITLE
When is layer freezing going to be enabled in yolov8?

### DISCUSSION THREAD

**Discussion by zz4fap** [ultralytics/ultralytics#2153] [General]
Dear, All!

When is layer freezing going to be enabled in yolov8?
We have tried the CLI "freeze" command, but it was to no avail.

Is there another way to freeze layers? 

Thanks and Kind Regards,

Felipe

**Comment by glenn-jocher** [comment]
@zz4fap hello Felipe,

Layer freezing is not currently supported in the YOLOv8 CLI as a standalone command. However, the functionality to freeze layers can be achieved during the training process by modifying the model configuration file. You would need to adjust the parameters to specify which layers to freeze.

For detailed guidance on how to do this, please refer to our documentation at https://docs.ultralytics.com. If you have further questions or need assistance with the configuration, feel free to open an issue in the repo, and we'll be happy to help!
